### DISCUSSION TITLE
Restore a Mesh Central backup from Linux to Windows

### DISCUSSION THREAD

**Discussion by rrmoore64** [Ylianst/MeshCentral#6615] [General]
I would like to migrate my Mesh Central server from a Linux (Ubuntu 22.04) instance that has been running about two years to a Windows server.  Should I be able to just restore the backup file from the linux server to the Windows hosted one?

**Comment by si458** [comment]
Yes :)

Just copy over the whole meshcentral-data folder from one to over and away u go :)

**Comment by rrmoore64** [comment]
This worked - although I did have to remember to work on the certificates, etc.  

Now, if we can get some momentum going again on Mac clients - that would be terrific!  All my organizations are moving to 100% MacOS.

Happy Holidays to all - best wishes

**Reply by si458** [reply]
well unfortunately unless ur company is willing to help us 

1. purchase a new apple silicon mac of some kind,
2. apple developer program plan,
3. employ someone who knows mac and can have us fix the screencapturekit bugs

its going to be a very long wait im afraid!
i would advise ur company to stay away from sequoia and stick to sonoma for now until we can fix the mac agent

**Reply by rrmoore64** [reply]
The first two could be a possibility.  The third is where we'd run into a block.  My coding days are far behind me now - and were all C and Windows.  We'll leave it for another appropriate thread!

**Reply by si458** [reply]
sure no worries, if u wanted to discuss anything like option 1 or 2,
do email myself (check my github page for email) or si458 on all chat platforms generally!
i myself do have a tiny bit of mac knowledge but its not perfect and i could end up spending weeks just trying to get a simple sample working haha
